src/firebase/models/players_in_room.js

```diff
@@ -0,0 +1,34 @@
+import { ref, onValue, onDisconnect, set, child } from "firebase/database";
+import rtdb from "../rtdb";
+
+const rooms = ref(rtdb, "rooms");
```

Comment:
Establish rooms as a variable which a reference to the rooms collection in the realtime database